scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/data/TaskProActiveDataspaces.java

```diff
@@ -572,9 +631,11 @@ private String virtualResolve(DataSpacesFileObject dataSpacesFileObject) {
         }
     }
 
-    private FileSystemException findFilesToCopyFromInputToScratch(List<InputSelector> inputSelectors,
-            ArrayList<DataSpacesFileObject> inResults, ArrayList<DataSpacesFileObject> outResults,
-            ArrayList<DataSpacesFileObject> globResults, ArrayList<DataSpacesFileObject> userResults) {
+    private FileSystemException findFilesToCopyFromInput(List<InputSelector> inputSelectors,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 9 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00107)